kotlinx.fuzz.api/src/main/kotlin/kotlinx/fuzz/KFuzzConfig.kt

```diff
@@ -99,6 +118,10 @@ class KFuzzConfigImpl private constructor() : KFuzzConfig {
     }
 }
 
+enum class RunMode {
+    FUZZING, REGRESSION, REGRESSION_FUZZING
```

Comment:
Right now, looking on the further code, `REGRESSION_FUZZING` seems more like `FUZZING + REGRESSION` rather than something compeletely different. 

So, the question is do we want to change `runMode: RunMode` to `runModes: List<RunMode>`? On the one hand, if we won't add new `RunModes` then current solution is totally fine. On the other hand, if yes, this change will make it much easier. 
@AbdullinAM what do you think?

Reply:
Fixed